ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopicBackupImpl.java

```diff
@@ -120,8 +120,23 @@ private boolean isRestoring() {
 
   private boolean isRecordInLatestReplay(final ConsumerRecord<CommandId, Command> record) {
     final Pair<CommandId, Command> latestReplayRecord = latestReplay.get(latestReplayIdx);
```

Comment:
why are we dealing with the deserialized command and commandid? `CommandTopic` reads the records as byte[]. Why can't we just back up exactly what we read. I understand deserializing before backing up as a sanity check, but we can still write the raw byte[] from the actual backup operation.

Reply:
Are the raw bytes UTF8 encoded? What you suggest might be a really good solution. The only thing that's important to me is to make sure the backup file is human readable (which it might be if we just write the bytes of what we read anyway)

Reply:
Yep, it should just be a json string.

Reply:
💯 I like your suggestion best then

Reply:
Using this deserialization in the past was unnecessary, but I just detected while fixing this PR. I initially was going with this approach of writing the string directly, but I wanted to keep things compatible with old backups. However, if the new proposal is to change the location in cc-spec-ksql to ignore them, then I'll remove the unnecessary code and write bytes directly.